tests/Backend/Workspaces/LegacyWorkspacesLoadTest.php

```diff
@@ -82,15 +82,7 @@ public function testWorkspaceLoadData()
 
         $workspaces->loadWorkspaceData($workspace['id'], array("input" => $input));
 
-        $afterJobs = array_filter(
-            $this->_client->listJobs(),
-            function ($job) use ($runId) {
-                return $job['runId'] === $runId;
-            }
-        );
-
-        $this->assertCount(1, $afterJobs);
```

Comment:
Tady ten assert ti zmizel.

Reply:
Nemá to vliv. A až budeme reusovat workspacy nebo mít možní async z akládání tak to bude i zbytečné.